### PR TITLE
Make generate script runable from anywhere

### PR DESCRIPTION
**Pull request by jomarko**
With these changes, user should be able to use all these ways of invoking scripts:
- `<sandbox-ui-root>$ npm run fetch-openapi`
- `<sandbox-ui-root>$ scripts/fetch_openapi.sh`
- `<random-dir>$ path1/path2/sandbox-ui/scripts/fetch_openapi.sh`

The same for generating.

The changes in PR
- Simplified and moved `.openapi-generator-ignore`. According to the documentation it should be in the output directory https://github.com/OpenAPITools/openapi-generator/blob/master/docs/customization.md#ignore-file-format
- `$(dirname $0)` used in generate script to follow approach from `fetch`
- Added comment into downloaded `smart-events.yaml` to make clear, it was downloaded, as we do not have it in `generated` directory

Honestly, not script expert, not sure if `package_name=$(dirname $0)/../smart-events` is needed or `package_name="smart-events"` is enough.

### CODE REVIEW

File: openapi/generated/.openapi-generator-ignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package.json
+tsconfig.json
+.npmignore
+.gitignore
+README.md
+.travis.yml
+.openapi-generator-ignore
+.openapi-generator
+git_push.sh
```
Comment (kelvah) [inline openapi/generated/.openapi-generator-ignore:9]: You should probably remove the `--ignore-file-override=.openapi-generator-ignore` parameter in the [generation script](https://github.com/jomarko/sandbox-ui/blob/PR-MGDOBR-420/scripts/generate_openapi_client.sh#L30) if the ignore file is moved to the output folder. Right?

File: scripts/fetch_openapi.sh [hunk 2]
```diff
@@ -10,3 +10,4 @@ if [ $? != 0 ]; then
   exit $?
 fi
 
+echo -e "# This file was downloaded automatically by 'fetch_openapi.sh'.\n# Do not edit this file manually!\n$(cat $OPENAPI_FILE_NAME)" > $OPENAPI_FILE_NAME
```
Comment (kelvah) [inline scripts/fetch_openapi.sh:13]: :+1:

File: scripts/generate_openapi_client.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-openapi_filename="openapi/smart-events.yaml"
-output_path="openapi/generated"
-package_name="smart-events"
+openapi_filename=$(dirname $0)/../openapi/smart-events.yaml
+output_path=$(dirname $0)/../openapi/generated
+package_name=$(dirname $0)/../smart-events
```
Comment (kelvah) [inline scripts/generate_openapi_client.sh]: Like you said I think the package name should just be `smart-events`.

**Review by kelvah — COMMENTED** [review]
Thank you for improving things around!

File: openapi/generated/.openapi-generator-ignore (hunk 1 above)
Comment (jomarko) [reply openapi/generated/.openapi-generator-ignore:9]: @kelvah good point, will do

File: scripts/generate_openapi_client.sh (hunk 3 above)
Comment (jomarko) [reply scripts/generate_openapi_client.sh]: ok, I will do

**Comment by jomarko** [comment]
Changes pushed

**Pull request merged**